nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -848,6 +848,27 @@ def unittest_coverage(context):
     run_command(context, command)
 
 
+@task(pre=[unittest])
+def unittest_xml_coverage(context):
+    """Produce an XML coverage report that can be ingested into other tools."""
+    produce_xml_cmd = "coverage xml --include '{{ cookiecutter.app_name }}/*' --omit '*/migrations/*,*/tests/*'"
+    run_command(context, produce_xml_cmd)
+
+    # Since the container runs from a different working dir,
+    # We have to overwrite it with the current directory to play nice with native tools.
+    cur_dir = os.path.dirname(__file__)
+    cov_xml_path = os.path.join(cur_dir, "coverage.xml")
+
+    # I was originally going to use ET lib but linter throws a CVE warning
```

Comment:
Should be fine to use ElementTree and just add an appropriate `noqa` comment, IMHO.

Reply:
Okay latest commit i went back to using ET